op-e2e/bindings/l1block.go

```diff
@@ -26,12 +26,13 @@ var (
 	_ = common.Big1
```

Comment:
Had we just neglected to update this recently? And I guess we have no CI enforcement that it stays in sync?

Reply:
Correct. It only needs to be updated if a new function wants to be accessed in a test.

Ideally we'd converge on a single type of contract bindings in the monorepo, it's currently a mess. But I see this out of scope of this PR.